app/Http/Controllers/Actions/Approvals/ApproveApproval.php

```
@@ -18,30 +23,36 @@
-                    $model->notify(new RequestProposalForApprovedNotification(auth()->user()->token, $model));
-                    break;
-                default:
-                    break;
+            $notificationMap = [
+                ApprovalModels::PROJECT_PROPOSAL_REQUEST->name => RequestProposalForApprovedNotification::class,
+                ApprovalModels::PROJECT_CHANGE_REQUEST->name => ChangeRequestApprovedNotification::class,
+            ];
+            if (isset($notificationMap[$modelType])) {
+                $model->created_by_user->notify(new $notificationMap[$modelType]($request->bearerToken(), $model));
             }
         }
```

PHPMD (2.15.0)

The method approve uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)